soc/arm/arm/designstart/Kconfig.defconfig.cortex_m1

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Henrik Brix Andersen <henrik@brixandersen.dk>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_ARM_DESIGNSTART_FPGA_CORTEX_M1
+
+config SOC
+	default "cortex_m1"
```

Comment:
This could be designstart_cortex_m1, to make it more specific

Reply:
Fixed.